### PR TITLE
Adding post coments deletion feature at issue2 page

### PR DESCRIPTION
**Pull request by JisanAR03**
Fixed issue #1469 
Work:
    1.create a path for call the deletion request.(urls.py --- at website folder)
    2.create a function which return the comment data after delete the specific comment .(views.py --- at website folder)
    3.Write some jQuery code for make the ajax call(comments2.html --- at templates folder)
screenshot :
image

### CODE REVIEW

File: website/views.py [hunk 1]
```diff
@@ -2047,6 +2047,24 @@ def update_comment(request, issue_pk, comment_pk):
     }
     return render(request, "comments2.html",context)
     
+def delete_comment(request):
+    int_issue_pk = int(request.POST['issue_pk'])
+    issue = Issue.objects.get(pk=int_issue_pk)
+    if request.method == "POST":
+        all_comment = Comment.objects.filter(issue=issue)
+        comment = Comment.objects.get(pk=int(request.POST['comment_pk']))
```
Comment (DonnieBLT) [inline website/views.py:2055]: Can you add the user to the query?

**Review by DonnieBLT — CHANGES_REQUESTED** [review]

File: website/views.py (hunk 1 above)
Comment (JisanAR03) [reply website/views.py:2055]: > Can you add the user to the query?

yes sir sure.

File: website/views.py [hunk 2]
```diff
@@ -2047,6 +2047,19 @@ def update_comment(request, issue_pk, comment_pk):
     }
     return render(request, "comments2.html",context)
     
+def delete_comment(request):
+    int_issue_pk = int(request.POST['issue_pk'])
+    issue = Issue.objects.get(pk=int_issue_pk)
+    if request.method == "POST":
+        all_comment = Comment.objects.filter(issue=issue)
+        comment = Comment.objects.get(pk=int(request.POST['comment_pk']),author=request.user.username)
+        if request.user.username == comment.author:
```
Comment (DonnieBLT) [inline website/views.py:2056]: Can remove this line as the query verifies it.

File: website/views.py [hunk 3]
```diff
@@ -2047,6 +2047,19 @@ def update_comment(request, issue_pk, comment_pk):
     }
     return render(request, "comments2.html",context)
     
+def delete_comment(request):
+    int_issue_pk = int(request.POST['issue_pk'])
+    issue = Issue.objects.get(pk=int_issue_pk)
+    if request.method == "POST":
+        all_comment = Comment.objects.filter(issue=issue)
```
Comment (DonnieBLT) [inline website/views.py:2054]: All comment is loaded below so you can remove this too

**Review by DonnieBLT — CHANGES_REQUESTED** [review]

File: website/views.py (hunk 3 above)
Comment (JisanAR03) [reply website/views.py:2054]: "all_comment" for sir fetch whole comment is web page after delete the comment

**Review by DonnieBLT — APPROVED** [review]

**Review by DonnieBLT — APPROVED** [review]

**Pull request merged**